packages/libllvm/tools-clang-lib-Driver-ToolChains-Linux.cpp.patch

```diff
@@ -1,18 +1,41 @@
---- ./tools/clang/lib/Driver/ToolChains/Linux.cpp.orig	2018-07-30 19:44:13.000000000 +0000
-+++ ./tools/clang/lib/Driver/ToolChains/Linux.cpp	2019-03-14 03:42:40.783434246 +0000
-@@ -291,7 +291,10 @@
+--- /home/builder/Linux.cpp.orig	2019-03-16 21:28:28.549998238 +0000
++++ ./tools/clang/lib/Driver/ToolChains/Linux.cpp	2019-03-16 23:43:01.266771835 +0000
+@@ -291,7 +291,33 @@
    // possible permutations of these directories, and seeing which ones it added
    // to the link paths.
    path_list &Paths = getFilePaths();
 -
 +  if(IsAndroid) {
-+    addPathIfExists(D, SysRoot + "@TERMUX_PREFIX@/lib", Paths);
-+	ExtraOpts.push_back("-rpath=@TERMUX_PREFIX@/lib");
++    if (Arch == llvm::Triple::arm || Arch == llvm::Triple::thumb) {
++      addPathIfExists(D, SysRoot + "@TERMUX_PREFIX@/arm-linux-androideabi/lib", Paths);
++      addPathIfExists(D, SysRoot + "@TERMUX_PREFIX@/lib", Paths);
```

Comment:
Already added above, same for next three.

Reply:
? 
rpath and the addpathifexists are different things 
addpath is search path as in -L/system/lib 
rpath is the search path it for libraries that the binary tells the linker where to search for them. 
the order of of search paths is important. The order of rpath is also important. they are 2 different things? am i missing something obvious?

Reply:
the only thing iwould change at this point is to have it not use multi link search paths or rpaths for native compilation.

Reply:
Ignore my review comments. I was confused because on mobile this diff shows the two lines you removed as mostly green with + symbols, so I thought you were adding those two lines too. Now that I viewed this diff in a non-mobile view, seems fine.